drone-fly-integration-tests/src/test/java/com/expediagroup/dataplatform/dronefly/core/integration/DroneFlyIntegrationTest.java

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright (C) 2020 Expedia, Inc.
```

Comment:
Not for this PR, but just noticed that we should probably update this to say `Expedia Group` in the oss parent

Reply:
No, we shouldn't. The last time I checked with the legal team the above is still correct.

Reply:
oh really? that is confusing 🤷🏼‍♀️